src/App.jsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import './App.scss';
 
+import { Person } from './components/Person/Person';
```

Comment:
remove unnecessary lines
```suggestion
import { Person } from './components/Person/Person';
```